src/store/ChartStore.ts

```diff
@@ -1051,10 +1048,10 @@ class ChartStore {
         }
     }
     calculateYaxisWidth = (price: number) => {
-        if (!price) return;
+        if (!price || this.isBarrierDragging) return;
 
         const { context } = this.context?.stx.chart;
-        const priceWidth = context.measureText(price.toFixed(this.pip)).width + 20;
+        const priceWidth = context.measureText(price.toString()).width + 20;
```

Comment:
Have you checked whether this works as expected with different currencies/markets and trade types? I believe depending on these parameters, the amount of decimal points differs, that's why there was a function to fix it.

Reply:
I checked few of others.. I'll ask qa to check more detailed. without fixing it we will be unable to adjust yaxis according to new barriers values